glib/src/auto/functions.rs

```diff
@@ -293,6 +293,34 @@ pub fn filename_from_uri(
     }
 }
 
+#[doc(alias = "g_filename_from_utf8")]
+pub fn filename_from_utf8(
+    utf8string: &str,
+) -> Result<(std::path::PathBuf, usize, usize), crate::Error> {
+    let len = utf8string.len() as isize;
+    unsafe {
+        let mut bytes_read = mem::MaybeUninit::uninit();
+        let mut bytes_written = mem::MaybeUninit::uninit();
+        let mut error = ptr::null_mut();
+        let ret = ffi::g_filename_from_utf8(
+            utf8string.to_glib_none().0,
+            len,
+            bytes_read.as_mut_ptr(),
+            bytes_written.as_mut_ptr(),
+            &mut error,
+        );
+        if error.is_null() {
+            Ok((
+                from_glib_full(ret),
+                bytes_read.assume_init(),
+                bytes_written.assume_init(),
```

Comment:
Isn't this equivalent to the length of the returned `PathBuf`?

Reply:
Yes, `FromGlibContainer` is now implemented for `PathBuf` so the scan for the nul byte can be avoided